### ISSUE TITLE
Why are certain keywords only allowed in certain option scopes?

### ISSUE THREAD

**Issue by mapfiable** [plk/biblatex#1334]
I'm sure this has been asked before, but I couldn't find an answer anywhere. 

In my case for example, I was looking for a way to change the `maxnames` keyword in a local `refsection`, but apparently, I can only do so on a per-entry basis. Now this isn't super tedious in my case, but I can see it quickly leading to issues with larger bibliographies. It would be much nicer to set these things on a local scale and only once in one place.

**Comment by plk** [comment]
The reason it's not there is that it's a really niche requirement - very few people want to change this per refsection (a lot of major styles like APA use the `uniquelist` option which is basically a dynamic `maxnames`). You can likely get what you want more easily by setting it as a per-type option which would mean you wouldn't have change your `.bib`.

**Comment by mapfiable** [comment]
Ah, I see. Thank you. I thought maybe it's difficult to implement for certain keywords, but I get that it's just due to low priority.

**Comment by mapfiable** [comment]
Ah, I see. Thank you. I thought maybe it's difficult to implement for certain keywords, but I get that it's just due to low priority.
